provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -258,6 +258,12 @@ def _create_node(self, machine_id: str) -> NodeDesc:
         hostname = store.get(f'server_node>{machine_id}>hostname')
         name = store.get(f'server_node>{machine_id}>name')
         iface = self._get_iface(machine_id)
+        try:
+            no_m0clients = int(store.get(
+                f'server_node>{machine_id}>client_instances',
```

Comment:
@supriyachavan4398 , conf-store key `cortx>software>motr>service>client_instances` is different in https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys and the change is being tracked by https://jts.seagate.com/browse/EOS-21908, which is not compete yet. Then why are we addressing this issue now?
Even the old key given in the ticket is different than what we are using here. I think atleast change this one to the old key as mentioned in the confluence page or differ this change until EOS-21908 is fixed. cc @vaibhavparatwar

Reply:
@mssawant , I looked at your comment and made changes accordingly. Now I used new Key `cortx>software>motr>service>client_instances` for that.